README.md

```diff
@@ -2,112 +2,157 @@
-7. Click the **Get Started** button to connect your GitHub account.
+2. From the top navigation bar in Jira, select **Apps > Find new Apps**. You can also click the **Gear icon** in 
+   the top-right corner and select **Apps**. 
+3. Search for **GitHub for Jira** and select **Get app**
+4. When the app is installed, a flag will pop up in the top-right corner. Click **Get Started** to connect your GitHub 
+   account. If you missed this flag, click **Configure integration** from the Apps menu.
+
+Next, you will need to [configure the app](#configure-app).
+
+## Install from Atlassian Marketplace
 
-Next you will need to [connect your GitHub organization to Jira](#connecting-your-github-organization-to-jira).
+1. Go to [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1219592/github-for-jira?hosting=cloud&tab=overview).
+2. Sign in to your Atlassian account and click **Get it now**.
+3. Select your site to install this app, click **Install app**.
+4. You will be taken to the app listing on your Jira site, click **Get app**.
+5. When the app is installed, a flag will pop up in the top-right corner. Click **Get Started** to connect your GitHub 
+   account. If you missed this flag, click **Configure integration** from the Apps menu.
 
-### Installation from GitHub Marketplace
+Next, you will need to [configure the app](#configure-app).
+
+### Install from GitHub Marketplace
 1. Go to https://github.com/marketplace/jira-software-github.
 2. Complete the (free) order for your GitHub Organization.
-3. On the installation settings screen, choose which repositories you want to use with the Jira Integration and press **Install**.
-4. Once installation completes, if we can't detect your Jira site, you will be prompted to enter a url, otherwise you will be redirected to the Jira marketplace to install the GitHub for Jira app.
-5. When you reach the page in marketplace, simply click **Get Started**.
+3. Choose which repositories you want to use with the Jira Integration on the installation settings screen, and click 
+   **Install**.
+4. You will be directed to a setup page to select your Jira site.
+5. Once installation completes you will be redirected to [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1219592/github-for-jira?hosting=cloud&tab=overview) to install the GitHub for Jira app.
+6. From there, follow the instructions above to [install from Atlassian Marketplace](#install-from-atlassian-marketplace).
+
+## Configure app
+
+### Connect your GitHub organization to Jira
+As part of the installation flow, you should be directed to a **GitHub configuration** screen to configure your GitHub 
+organizations in the Jira app.
+
+> :warning: If you are not directed, navigate to the Apps page of your Jira instance and click **Configure integration** under the ”GitHub” heading. If you do not see this, click on **Manage apps** and **Get started** under GitHub for Jira. This will bring you to the app's configuration screen.
```

Comment:
Maybe it would be good to state here that only Jira admins can see the `Manage your apps` link.

Reply:
nit: apparently it changed from `Manage apps` to `Manage your apps`
<img width="334" alt="Screen Shot 2021-11-16 at 9 18 01 am" src="https://user-images.githubusercontent.com/58801956/141861735-179ea0d7-02bf-4d4a-8dc4-223ff8e12c6b.png">